examples/bigquery-table-access-pattern-analysis/README.md

```diff
@@ -0,0 +1,419 @@
+# Table Access Pattern Analysis
+This module consists of deep dive analysis of a BigQuery environment in Google Cloud Platform, according to audt logs - data access data, which can be used to optimise BigQuery usage, and improve time, space and cost of BigQuery.
+
+## Pipeline Optimisation
+
+#### <b>Definitions</b>
+The word <b>'pipeline'</b> here refers to one step of the transformation process in data warehouse, in this case BigQuery, as part of an ELT job. Each pipeline involves source table(s) and destination table. For example, a query job 
```

Comment:
To explain your usage of pipeline better I think you mean:

The word 'pipeline' refers to a single query in the transformation process in a data warehouse, in this case BigQuery. We are talking collectively about all instances of this query. Each "pipeline" involves source table(s) and destination table. For example, a query:

Reply:
Thanks, is this better?

The word 'pipeline' refers to a collection of all job instances of a query in the transformation process in a data warehouse, in this case BigQuery. Each "pipeline" involves source table(s) and destination table. For example, a query:

because I think 
'The word 'pipeline' refers to a single query in the transformation process' is a bit ambiguous, people might interpret it as a single job, instead of a single query

Reply:
yep that's better